pvt/helpers/src/models/pools/weighted/types.ts

```diff
@@ -8,14 +8,6 @@ import TokenList from '../../tokens/TokenList';
 import { Account, NAry } from '../../types/types';
 import Vault from '../../vault/Vault';
 
-export enum WeightedPoolType {
```

Comment:
given that we'll only use this to select different kinds of managed pools (considering mock settings as one of them), maybe the enum can be recycled as `ManagedPoolType` and just be used for the managed pool deployment (by default, `MANAGED_POOL`).